src/TypedSignalR.Client.TypeScript/SpecialSymbols.cs

```diff
@@ -20,23 +20,8 @@ public SpecialSymbols(Compilation compilation)
         var receiverAttributeSymbol = compilation.GetTypeByMetadataName("TypedSignalR.Client.ReceiverAttribute");
         var transpilationSourceAttributeSymbol = compilation.GetTypeByMetadataName("Tapper.TranspilationSourceAttribute");
 
-        if (hubAttributeSymbol is null)
-        {
-            throw new InvalidOperationException("TypedSignalR.Client.HubAttribute is not found");
-        }
-
-        if (receiverAttributeSymbol is null)
-        {
-            throw new InvalidOperationException("TypedSignalR.Client.ReceiverAttribute is not found");
-        }
-
-        if (transpilationSourceAttributeSymbol is null)
-        {
-            throw new InvalidOperationException("Tapper.TranspilationSourceAttribute is not found");
-        }
-
-        HubAttributeSymbol = hubAttributeSymbol;
-        ReceiverAttributeSymbol = receiverAttributeSymbol;
-        TranspilationSourceAttributeSymbol = transpilationSourceAttributeSymbol;
+        HubAttributeSymbol = hubAttributeSymbol ?? throw new InvalidOperationException("TypedSignalR.Client.HubAttribute is not found");
```

Comment:
Don't use a null-coalescing operator to throw Exception.

Reply:
Reverted